src/main/java/uk/gov/companieshouse/acsp/service/FilingsService.java

```diff
@@ -324,6 +324,9 @@ private Address buildRegisteredOfficeAddress(AcspDataDto acspDataDto) {
         registeredOfficeAddress.setRegion(
               Optional.ofNullable(acspDataDto.getRegisteredOfficeAddress().getRegion())
                       .map(String::toUpperCase).orElse(null));
+      registeredOfficeAddress.setLocality(
```

Comment:
Is setting locality not required in buildCorrespondenAddress method ?

Reply:
this a test fix for the mapping mismatch we got in the registered address xml generated file, if it works good and fix the issue I will update the corrAddress after